lib-content-scrapers/src/jvmMain/kotlin/com/ustadmobile/lib/contentscrapers/folder/FileScraper.kt

```diff
@@ -28,7 +29,7 @@ class FileScraper(contentEntryUid: Long, sqiUid: Int, parentContentEntryUid: Lon
 
             try {
 
-                val metadata = extractContentEntryMetadataFromFile(file, db)
+                val metadata = contentImportManager.extractMetadata(file.path)
 
                 if (metadata == null) {
```

Comment:
This isn't directly part of this commit or objective, but for scrapers we want to avoid inserting a contententry until after everything is validated and ready to go, such that we don't have to hide things afterwards.

Reply:
metadata doesnt insert contentEntry